src/app/questionForMode.js

```diff
@@ -0,0 +1,11 @@
+export const questionForMode = (mode) => {
+  const questions = {
+    People: 'Who is this character?',
+    Houses: 'What is this house?',
+    Wands: 'Who this wand belongs to?',
+  };
+
+  const modeQuestion = `MODE: ${questions[mode]}`;
+
+  document.querySelector('.game__mode').textContent = modeQuestion;
```

Comment:
The part of the task is also styling the mode question, perhaps you could give it a thought? It is a container with text abd this text might be the only thing about it that is different from other (maybe its a specfic heading), so perhaps you should also think of it not as a single mode question, but one of a group of containers, so you could reuse the same styling for mode rules or maybe even buttons? Something to think about.